group_vars/all.yml

```diff
@@ -14,20 +14,18 @@ default_node_types:
   - validator
   - default
 
-sui:
+tmp_dir: /tmp
+
+chainflip:
   validator:
-    source_url: https://releases.sui.io
+    source_url: https://github.com/chainflip-io/chainflip-backend
     ports:
-      server_listen_port: 9191
-      metrics_port: 9186
+      p2p_port: 30333
+      rpc_port: 9944
```

Comment:
we might to have change this, I think in avail we might have used the same ports